lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -237,11 +238,17 @@ def race(races: str or list) -> list:
 
     race_map = {
         "American Indian or Alaska Native": "americanIndianOrAlaskaNative",
+        "amerind": "americanIndianOrAlaskaNative",
         "Asian": "asian",
+        "asian": "asian",
```

Comment:
This seems a little redundant. Would it be better to check `if race[i] in race_map.values()` then return `race[i]`.

Reply:
Fair point. I now check `if races[i] in race_map.values()` and `continue` if true. 

I can't return `race[i]` though because I need to handle multiple races.